managed/ui/src/components/config/Security/KeyManagementConfigurationContainer.js

```diff
@@ -12,13 +12,15 @@ import {
   deleteKMSProviderConfigResponse
 } from '../../../actions/cloud';
 import { toast } from 'react-toastify';
+import { closeDialog, openDialog } from '../../../actions/modal';
 
 const mapStateToProps = (state) => {
   return {
     customerConfigs: state.customer.configs,
     configList: state.cloud.authConfig,
     visibleModal: state.modal.visibleModal,
-    deleteConfig: state.customer.deleteConfig
+    deleteConfig: state.customer.deleteConfig,
+    modal: state.modal
```

Comment:
I guess since you are using local state - the `modal` prop from store is not needed anymore